File: 2020/aoc01.py
input_text = open('input.txt',)
ll = input_text.readlines()
for l in ll:
    for k in ll:
        if int(l)+int(k) == 2020:
            print(int(l)*int(k))

for l in ll:
    for k in ll:
        for j in ll:
            if int(l)+int(k)+int(j) == 2020:
                print(int(l)*int(k)*int(j))

File: 2023/aoc01.c
#include <stdio.h>

void main() {
  FILE *fptr;
  
  fptr = fopen("input.txt", "r");
  
  char myString[100];
  long ans = 0;
  
  while(fgets(myString, 100, fptr)) {
    int i;
    char s = -1;
    char e;
    char *ptr;
    for (i=0; i<strlen(myString); i++) {
      if (isdigit(myString[i])) {
        if (s == -1) {
          s = myString[i];
        }
        e = myString[i];
      }
    }
    char out[] = "12";
    out[0] = s;
    out[1] = e;
    long ret = strtol(out,&ptr,10);
    ans += ret;
  }
  printf("%ld", ans);
  fclose(fptr);
}